explorer/src/config.ts

```diff
@@ -10,6 +10,10 @@ export interface ExplorerConfig {
    * Whether dev mode is enabled or not
    */
   dev: boolean
```

Comment:
Might be better to model this as an enumeration of Dev | Production to avoid getting into dev and prod at the same time.